shell/common/engine.cc

```diff
@@ -174,7 +175,8 @@ fml::WeakPtr<ImageGeneratorRegistry> Engine::GetImageGeneratorRegistry() {
 
 bool Engine::UpdateAssetManager(
     const std::shared_ptr<AssetManager>& new_asset_manager) {
-  if (asset_manager_ == new_asset_manager) {
+  if ((asset_manager_ && new_asset_manager) &&
```

Comment:
nit: the parentheses around `(asset_manager_ && new_asset_manager)` are not needed

Reply:
Done